onnx/reference/custom_element_types.py

```diff
@@ -1,14 +1,20 @@
 # Copyright (c) ONNX Project Contributors
-#
+
 # SPDX-License-Identifier: Apache-2.0
 from __future__ import annotations
 
-import numpy as np
+import warnings
+
+from onnx.custom_element_types import bfloat16  # noqa: F401
```

Comment:
It’s possible to define `__all__` and fix f401

Reply:
done